gitops/components/multus-service/kustomization.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+resources:
+  - ./resources.yaml
+
+patches:
+  - target:
+      group: apps
+      version: v1
+      kind: Deployment
+      name: multus-service-controller
+    path: ./overlays/multus-tolerations.yaml
```

Comment:
Is this intending to patch the `Deployment` with the same patch as applied to the `DaemonSet` below?

Reply:
yep there is a weird behavior around patches, metadata is required to run (in the patch file) but not used as a replacement OR the initial target set in `kustomization.yaml` is overriding it another time

Reply:
Clearly ugly + the `container/0/env/0` - if there is any better way to avoid potential future issues with this kind of syntax - also this file is a duplicate from Multus folder. Somehow not having to reference metadata and using for all the same patch would be nicer.